LEGO1/tgl/d3drm/something.cpp

```diff
@@ -0,0 +1,63 @@
+#include "impl.h"
+
+using namespace TglImpl;
+
+DECOMP_SIZE_ASSERT(Something, 0x4);
+DECOMP_SIZE_ASSERT(SomethingImpl, 0x8);
+
+// Inlined only
+SomethingImpl::~SomethingImpl()
```

Comment:
We generally use the term "Unknown" (or `Unk`) in the rest of the code base for unknown stuff, so we should apply it here as well

Reply:
In general, naming anything "Something" is pretty objectively bad practice. There are hundreds if not thousands of classes we don't know yet that could all theoretically be called "Something", but how does that help us identify them? If anything, it makes it _harder_ to distinguish them than if it was just offsets.

Reply:
There's only one unknown class in the Tgl namespace, the rest of the codebase explicitly uses Tgl classes as `Tgl::` so I don't think there's an issue with using an ambiguous name.

Though I'm not attached to "something", changed it to `Unk` (Using `Unknown` breaks code completion because Microsoft in their infinite wisdom decided to use `Unknown` as an enum item name)